data/json/npcs/NC_DEBUG.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "type" : "item_group",
+    "id" : "NC_DEBUG_worn",
+    "items":[ ["towel", 100] ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "NC_DEBUG_carried",
+    "items":[ ["mininuke", 100] ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "NC_DEBUG_weapon",
+    "items":[ ["soap", 100] ]
```

Comment:
I just wanted a random item to show that it works.
The joke only happened later, when the NPC invariably decides to drop said weapon on the first turn because it isn't a good weapon.